vpr/src/timing/VprTimingGraphResolver.cpp

```diff
@@ -324,18 +324,18 @@ void VprTimingGraphResolver::get_detailed_interconnect_components_helper(std::ve
 
                 arrow = "->"; //we will point the coordinates from start to finish, left to right
 
-                if (rr_graph.node_direction(RRNodeId(node->inode)) == Direction::DEC) {              //signal travels along decreasing direction
-                    start_x = " (" + std::to_string(device_ctx.rr_nodes[node->inode].xhigh()) + ","; //start coordinates have large value
-                    start_y = std::to_string(device_ctx.rr_nodes[node->inode].yhigh()) + ")";
-                    end_x = "(" + std::to_string(device_ctx.rr_nodes[node->inode].xlow()) + ","; //end coordinates have smaller value
-                    end_y = std::to_string(device_ctx.rr_nodes[node->inode].ylow()) + ")";
+                if (rr_graph.node_direction(RRNodeId(node->inode)) == Direction::DEC) {                //signal travels along decreasing direction
```

Comment:
We may consider to have an API ``node_coordinate_to_string()`` which prints the string for a given node (depending on the direction) in this format.

Reply:
This can definitely be done. @vaughnbetz what are your thoughts on adding this API?

Reply:
I like it (as we discussed today).  Use it at least in the VprTimingGraphResolver, and there may be other uses (error messages,, overused rrnodes when we fail to route etc.) that could be captured now, and likely more in the future.
So I'd add it, and do a quick search to see if you can find routines that should use it now instead of printing ad-hoc RRnode identifiers.